modules/seqexec/engine/src/main/scala/seqexec/engine/ActionCoords.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.engine
+
+import cats.Eq
+import cats.implicits._
+import gem.Observation
+import seqexec.model.StepId
+
+final case class ActionIndex(self: Long) extends AnyVal
+object ActionIndex {
+  implicit val actionIndexEq: Eq[ActionIndex] = Eq.by(_.self)
+}
+
+final case class ExecutionIndex(self: Long) extends AnyVal
+object ExecutionIndex {
+  implicit val actionIndexEq: Eq[ExecutionIndex] = Eq.by(_.self)
+}
+
+final case class ActionCoordsInSeq(stepId: StepId, execIdx: ExecutionIndex,
```

Comment:
Somehow I'd expect the resource to be here. Like we want to execute `TCS` on step `stepId`?

Reply:
Resource control is in the level above Engine

Reply:
Resource control is in the level above Engine

Reply:
Resource control is in the level above Engine